pkgs/games/katago/default.nix

```
@@ -67,25 +68,27 @@ stdenv.mkDerivation rec {
   ];
 
   cmakeFlags = [
-    "-DNO_GIT_REVISION=ON"
+    (lib.cmakeBool "NO_GIT_REVISION" true)
   ] ++ lib.optionals enableAVX2 [
-    "-DUSE_AVX2=ON"
+    (lib.cmakeBool "USE_AVX2" true)
```

Maybe
```suggestion
    (lib.cmakeBool "USE_AVX2" enableAVX2)
```
?

(and remove the `lib.optionals` since it will be redundant)